tests/vcs/git/GitCommitBearTest.py

```diff
@@ -139,6 +139,17 @@ def test_shortlog_checks_shortlog_trailing_period(self):
         self.assertEqual(self.run_uut(shortlog_trailing_period=False), [])
         self.assertEqual(self.run_uut(shortlog_trailing_period=None), [])
 
+    def test_shortlog_checks_imperative(self):
+        self.git_commit("tag: Add shortlog in imperative")
+        self.assertNotIn("Shortlog of HEAD commit isn't imperative mood",
+                         ''.join(self.run_uut()))
+        self.git_commit("Added invalid shortlog")
+        self.assertIn("Shortlog of HEAD commit isn't imperative mood",
+                      ''.join(self.run_uut()))
+        self.git_commit("Adding another invalid shortlog")
+        self.assertIn("Shortlog of HEAD commit isn't imperative mood",
+                      ''.join(self.run_uut()))
```

Comment:
Ah sadly nltk isn't that smart. It would classify "maximizing" in Present participle category. We can solve this problem by restricting users to follow certain grammar structure. Something like 
Verb | <anything>. So in that case we would test the first word only. Still it won't be perfect. For eg.
`tag: Add. This added stuff`. 
Clearly this ain't imperative. :D